api/document/serializers/content.py

```diff
@@ -92,7 +92,7 @@ def get_attribute(self,
 
 class InlinesField(NestableAnnotationField):
     def __init__(self, is_leaf_node: bool) -> None:
-        super().__init__()
+        super().__init__(default=lambda: [])
```

Comment:
I also went ahead and made the serializer default `inlines` to an empty list, so that it didn't expect it in silly cases (such as when `__text__` content, which doesn't even support inlines, is submitted).